frontend/packages/knative-plugin/src/components/add/event-sources/ApiServerSection.tsx

```diff
@@ -56,12 +56,13 @@ const ApiServerSection: React.FC<ApiServerSectionProps> = ({ title, fullWidth })
     }
   };
   return (
-    <FormSection title={title} extraMargin fullWidth={fullWidth}>
+    <FormSection title={title} extraMargin fullWidth={fullWidth} dataTest={`${title}-section`}>
```

Comment:
You maybe know, I would always recommend separate title and section with a space to enable search by `~=`.

```suggestion
    <FormSection title={title} extraMargin fullWidth={fullWidth} dataTest={`${title} section`}>
```

Reply:
Updated